server.js

```diff
@@ -89,16 +98,21 @@ io.on("connection", client => {
   // TODO Create a subscription for this new client to some occupancy changes
   // TODO subscribe to realOccupancyChanges AND simulatedOccupancyChanges
   // TODO subscribe to realOccupancyChanges AND simulatedOccupancyChanges
-  const sub = simulatedOccupancyChanges.subscribe(notifyClient);
+  const sub = merge(
+    simulatedOccupancyChanges,
+    roomHoldOccupancyChanges
+  ).subscribe(notifyClient);
```

Comment:
And here is where both streams/Observables of occupancy changes are combined into one, and the client is notified any time either of them notify.